ci/container_index/run.py

```diff
@@ -0,0 +1,115 @@
+import sys
+from os import path
+
+fl = path.dirname(path.realpath(__file__))
+sys.path.append(path.join(fl, "..", ".."))
```

Comment:
Add some comments here about what module needs to be added to the path.
Can we avoid this modification of sys.path in run.py and have the module path added at the starter script?  Like you can specify `PYTHONPATH` while invoking the CI initi.